pypsrp/wsman.py

```diff
@@ -773,6 +783,21 @@ def _build_session(self):
         elif self.no_proxy:
             session.proxies = orig_proxy
 
+        # Retry on connection errors, with a backoff factor
+        retries = requests.packages.urllib3.util.retry.Retry(
```

Comment:
I can't claim to know too much about requests but wouldn't this be an issue if urllib3 is not the requests backend? Maybe it would be better to make it an optional import.

Reply:
What other backends exist in requests ? To me it seems that requests is pretty much bolt onto urllib3.

Reply:
Well, the very first thing that happens in *requests/__init__.py* is `import urllib3`.
So I think we're safe ;-)